.github/workflows/deploy-to-environment.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         if: ${{ github.ref != 'refs/heads/main' }}
         run: |
           echo "This workflow can only be manually triggered with workflow_dispatch on the main branch"
-          exit 1
+          # exit 1
```

Comment:
Disabled for testing.  I actually believe that it should be possible to push any ref, not just head of main.  E.g. it should be possible to deploy a tagged release candidate to beta even if the tag is no longer at head of main.  But that is the subject of another PR.